dcos/package.py

```diff
@@ -943,6 +944,25 @@ def copy_to_cache(self, target_dir):
                 'Unable to fetch packages from [{}]'.format(self.url))
 
 
+def onerror(func, path, exc_info):
```

Comment:
This function for adding write permission to files on Windows platform. Because exception raised when trying to delete read-only files on Windows.